### PR TITLE
Add "Desenvolvimento de Aplicações Web em Java e Outras Tecnologias" to `pt_BR` books

### PR DESCRIPTION
**Pull request by davidbuzatto**
## What does this PR do?
Add resource

### Description
First edition of my book about basic web development using Java, writen in brazilian portuguese.

### Why is this valuable (or not)?
The book covers the very basics of Java Web development and other technologies related to it.

### How do we know it's really free?
The LaTeX source code of the book is available in GitHub (https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java) and the book was registered in ISBN.

### For book lists, is it a book? For course lists, is it a course? etc.
Book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -193,6 +193,7 @@
 
 ### Java
 
+* [Desenvolvimento de Aplicações Web em Java e Outras Tecnologias](https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases/download/v1.0ed-1.0c/DesenvolvimentoAplicacoesWebEmJava.-.1ed.-.final.com.correcoes.01.pdf) - David Buzatto
```
Comment (davorpa) [inline books/free-programming-books-pt_BR.md:196]: If you read Contributing guidelines... PDF resources needs mark it respective format

```suggestion
* [Desenvolvimento de Aplicações Web em Java e Outras Tecnologias](https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases/download/v1.0ed-1.0c/DesenvolvimentoAplicacoesWebEmJava.-.1ed.-.final.com.correcoes.01.pdf) - David Buzatto (PDF)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: books/free-programming-books-pt_BR.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-pt_BR.md:196]: I suggest using https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases so no updates are needed for new versions or added formats

Comment (davidbuzatto) [reply books/free-programming-books-pt_BR.md]: I changed the link to the project releases and added the PDF format. Thank you!

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**